GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -2264,6 +2268,14 @@ public void ToggleShowReflogReferences()
             OnFilterChanged();
         }
 
+        public void ToggleHideStashBranch()
```

Comment:
This only needs to be `public` if it is exposed in `IRevisionGridFilter` interface
```suggestion
        internal void ToggleHideStashBranch()
```